libs/designsystem/src/lib/components/radio/radio.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="wrapper">
+  <ion-radio
+    [value]="value"
+    [disabled]="disabled"
+    mode="md"
+    [attr.aria-labelledby]="text ? _labelId : null"
+  ></ion-radio>
+  <span [id]="_labelId" *ngIf="text">{{ text }}</span>
```

Comment:
consider using label instead of span due to accessibility?

Reply:
Ionics wraps it in a label ;)